extensions/amp-story/0.1/media-pool.js

```diff
@@ -92,6 +212,48 @@ function ampMediaElementFor(el) {
 }
 
 
+/**
+ * @param {!HTMLMediaElement} mediaEl The element whose task queue should be
+ *     executed.
+ */
+function executeNextMediaElementTask(mediaEl) {
+  const queue = mediaEl[ELEMENT_TASK_QUEUE_PROPERTY_NAME];
+
+  if (queue.length === 0) {
+    return;
+  }
+
+  const taskName = queue[0];
+  const task = ELEMENT_TASKS[taskName] || NOOP_ELEMENT_TASK;
+
+  task(mediaEl)
+      .catch(reason => dev().error('AMP-STORY', reason))
+      .then(() => {
```

Comment:
Cool pattern, but unless it's widely used, I think it's worth adding a ``// Finally`` comment

Reply:
done